libsplinter/src/biome/rest_api/auth/oauth.rs

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use uuid::Uuid;
+
+use crate::auth::{
+    oauth::{rest_api::SaveTokensOperation, UserTokens},
+    rest_api::identity::{Authorization, BearerToken, IdentityProvider},
+};
+use crate::biome::oauth::store::{OAuthProvider, OAuthUserBuilder, OAuthUserStore};
+use crate::biome::user::store::{User, UserStore};
+use crate::error::InternalError;
+
+#[derive(Clone)]
+pub struct OAuthUserStoreSaveTokensOperation {
```

Comment:
Missing doc comments throughout this module

Reply:
Fixed.